larpix/chip.py

```diff
@@ -75,7 +77,7 @@ def get_configuration_packets(self, packet_type, registers=None):
         packets = []
         if self.asic_version == 1:
             packet_register_data = enumerate(conf.all_data())
-        elif self.asic_version in (2, 'lightpix-v1.0'):
+        elif self.asic_version in (2, 'lightpix-v1.0', '2b', 'larpix-v2b'):
```

Comment:
Pick one, it is harder to maintain and less clear if we have multiple names floating around.